criu/kerndat.c

```diff
@@ -259,6 +260,33 @@ static int kerndat_get_shmemdev(void)
 	return -1;
 }
 
+static int kerndat_get_secretmem_dev(int fd)
+{
+	void *secretmem = NULL;
+	dev_t dev;
+
+	if (ftruncate(fd, PAGE_SIZE) < 0)
+		goto err;
+
+	secretmem = mmap(NULL, PAGE_SIZE, PROT_READ | PROT_WRITE, MAP_SHARED, fd, 0);
+	if (secretmem == MAP_FAILED)
+		return -1;
+
+	if (kerndat_get_dev(&dev, secretmem, PAGE_SIZE))
+		goto err;
+
+	munmap(secretmem, PAGE_SIZE);
+	kdat.secretmem_dev = dev;
+	pr_info("Found secret-memory device at %ld\n", kdat.secretmem_dev);
```

Comment:
See get_tty_driver and how PRIx64 is used to print dev_t, you should use it here to fix "Cross Compile Tests".

Reply:
done, thanks!